### DISCUSSION TITLE
official launcher requires eos. Please dont make it a requirement here too.

### DISCUSSION THREAD

**Discussion by ghost** [Heroic-Games-Launcher/HeroicGamesLauncher#1836] [General]
The latest version of the official epic games launcher now requires EOS to be installed, which previously was an optional install. epic games is so hellbent on having everyone install it that when the pop up automatically appears to install it, clicking X and alt f4 have no effect because they have been purposely disabled. https://gfycat.com/unawaresparsecalf

As HeroisGamesLauncher is an alternative to the highly invasive and anti consumer epicgames launcher, please do not make eos a requirement for HGL to function for adding and opening games. If the lack of eos breaks the multiplayer functionality of some games, still make it a optional install, please.

**Comment by flavioislima** [comment]
We have no intention of making this mandatory. 
Our main goal is to make Heroic runs with less jerk from epic as possible otherwise will stop making sense.
And like you said, if it becomes mandatory for some games we can add an install button just like we are doing with the Epic Overlay. so you can use it only on these specific games.
